### ISSUE TITLE
Add playground for testing retention and downsampling filters

### ISSUE THREAD

**Issue by hagen1778** [VictoriaMetrics/VictoriaMetrics#6304]
### Is your feature request related to a problem? Please describe

[Retention](https://docs.victoriametrics.com/#retention-filters) and [downsampling](https://docs.victoriametrics.com/#downsampling) filters setting could be complicated for environments with many retention policies. And user doesn't have a tool to test whether defined settings are correct for their needs and there is no way to test it.

### Describe the solution you'd like

Create a playground - a web UI page available in vmselect that would have the following inputs:
1. The list of time series. For example:
```
foo{vm_account_id="1"}
foo{vm_account_id="2"}
foo{vm_account_id="3"}
```
2. The list of configured retention filters as cmd-line flags:
```
-retentionPeriod=12
-retentionFilter='{vm_account_id="1"}:3
-retentionFilter='{vm_account_id=~"2|3"}:6
```

Once all inputs were filled, the playground can produce an output:
```
foo{vm_account_id="1"} - retention period is 3month
foo{vm_account_id="2"} - retention period is 6month
foo{vm_account_id="3"} - retention period is 6month
```

While this example is pretty simple and straightforward, there could be much more tricky cases with overlapping time series selectors. For example:
```
-retentionFilter='{vm_account_id="1"}:3
-retentionFilter='{vm_account_id!~"2|3"}:6
```
In this example user could have got unexpected results because second filter overlaps with the first filter. The result of the evaluation should produce the following:
```
foo{vm_account_id="1"} - retention period is 6month
foo{vm_account_id="2"} - retention period is 12month
foo{vm_account_id="3"} - retention period is 12month
```
and user could spot an error in the filters right away.

**Comment by f41gh7** [comment]
The bugfix was included into [v1.105.0](https://docs.victoriametrics.com/changelog/#v11050)

**Comment by hagen1778** [comment]
The new playground pages are included into vmselect's vmui in ENT version and available starting from v1.105.0.
They also can be visited on our playground:
* https://play.victoriametrics.com/select/accounting/1/6a716b0f-38bc-4856-90ce-448fd713e3fe/prometheus/graph/#/downsampling-filters-debug 
* https://play.victoriametrics.com/select/accounting/1/6a716b0f-38bc-4856-90ce-448fd713e3fe/prometheus/graph/#/retention-filters-debug
